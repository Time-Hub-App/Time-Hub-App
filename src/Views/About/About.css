body {
  font-family: 'UglyTypistRegular';
  font-weight: normal;
  font-style: normal;
}
