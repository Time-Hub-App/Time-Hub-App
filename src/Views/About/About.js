import './About.css';
import Header from '../../Components/Header/Header.js';
import Footer from '../../Components/Footer/Footer.js';

export default function About() {
  return (
    <>
      <Header />
      <h1>About Us</h1>
      <div>
        <h3>Fora Heims</h3>
        <ul>
          <p>foraheims@gmail.com</p>
          <p>Github: fora-heims</p>
        </ul>
        <p>bio</p>
      </div>
      <div>
        <h3>Libbi Dunham</h3>
        <ul>
          <p>Libbi.dunham25@gmail.com</p>
          <p>Github: Libbi-Dunham</p>
        </ul>
        <p>
          I am detailed oriented critical thinker who is switching into web development. Passion for
          learning new technical knowledge and being communitive with people.
        </p>
      </div>
      <div>
        <h3>Ckay Natividad</h3>
        <ul>
          <p>Natividad.ckay@gmail.com</p>
          <p>Github: ckaynatividad</p>
        </ul>
        <p>bio</p>
      </div>
      <div>
        <h3>Abriel Cleaver</h3>
        <ul>
          <p>abrielcleaver@gmail.com</p>
          <p>Github: abrielcleaver</p>
        </ul>
        <p>bio</p>
      </div>
      <Footer />
    </>
  );
}
