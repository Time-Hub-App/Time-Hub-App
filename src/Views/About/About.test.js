import { render } from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom';
import About from './About';

it('should render an about view component', () => {
  const { container } = render(
    <MemoryRouter>
      <About />
    </MemoryRouter>
  );

  expect(container).toMatchSnapshot();
});
