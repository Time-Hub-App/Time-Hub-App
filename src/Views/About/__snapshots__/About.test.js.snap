// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render an about view component 1`] = `
<div>
  <header>
    <h3
      class=""
    >
      Home
    </h3>
    <h3
      class=""
    >
      About
    </h3>
    <h3
      class=""
    >
      Log In
    </h3>
    <h3
      class=""
    >
      Journals
    </h3>
  </header>
  <h1>
    TimeHub
  </h1>
  <div
    class="timehub-bio"
  >
    <span>
      Journaling is a very underrated activity that boosts your well-being in just a few minutes a day!
      <br />
      A paper notebook and pen are fine for journaling, but TimeHub offers more! Jot your thoughts down or record what happened during the day for a simple way to manage stress, enhance creativity, increase happiness, and track your emotions.
    </span>
  </div>
  <div>
    <h2>
      Meet the Creators
    </h2>
  </div>
  <div
    class="developer-container"
  >
    <div
      class="developer-bio"
    >
      <h3
        class="name"
      >
        Fora Heims, Full Stack Software Developer
      </h3>
      <ul>
        <div>
          <a
            href="/https://www.linkedin.com/in/fora-heims/"
          >
            LinkedIn
          </a>
        </div>
        <div>
          <a
            href="/https://github.com/fora-heims"
          >
            GitHub
          </a>
        </div>
      </ul>
      <div>
        <p
          class="bio-text"
        >
          Fora is a dedicated, efficient and strategic developer. Because of their strong foundation of software related knowledge, Fora can produce productive, long-lasting, and clean code.
           
        </p>
      </div>
    </div>
    <div
      class="developer-bio"
    >
      <h3
        class="name"
      >
        Libbi Dunham, Full Stack Software Developer
      </h3>
      <ul>
        <div>
          <a
            href="/https://www.linkedin.com/in/libbi-dunham/"
          >
            LinkedIn
          </a>
        </div>
        <div>
          <a
            href="/https://github.com/Libbi-Dunham"
          >
            GitHub
          </a>
        </div>
      </ul>
      <p
        class="bio-text"
      >
        Libbi is a detailed oriented developer with critical thinking skills who is switching into web development. She has a passion for learning new technical knowledge and having open communication with her team.
      </p>
    </div>
    <div
      class="developer-bio2"
    >
      <h3
        class="name"
      >
        Ckay Natividad, Full Stack Software Developer
      </h3>
      <ul>
        <div>
          <a
            href="/https://www.linkedin.com/in/ckaynatividad/"
          >
            LinkedIn
          </a>
        </div>
        <div>
          <a
            href="/https://github.com/ckaynatividad"
          >
            GitHub
          </a>
        </div>
      </ul>
      <p
        class="bio-text2"
      >
        Ckay is a passionate developer with a versatile background ranging from visual arts to healthcare.
         
      </p>
    </div>
    <div
      class="developer-bio2"
    >
      <h3
        class="name"
      >
        Abriel Cleaver, Full Stack Software Developer
      </h3>
      <ul>
        <div>
          <a
            href="/https://www.linkedin.com/in/abrielcleaver/"
          >
            LinkedIn
          </a>
        </div>
        <div>
          <a
            href="/https://github.com/abrielcleaver"
          >
            GitHub
          </a>
        </div>
      </ul>
      <p
        class="bio-text2"
      >
        Abriel is a multifaceted developer with a background in Mental Health & Wellness, photography, and marketing who thrives when it comes to deconstructing an issue logically. She enjoys learning new technologies and working with diverse teams.
         
      </p>
    </div>
  </div>
  <footer>
    <a
      class="footer-link"
      href="/https://github.com/Time-Hub-App"
    >
      TimeHub GitHub
    </a>
  </footer>
</div>
`;
